common/test/acceptance/pages/studio/settings_advanced.py

```diff
@@ -266,8 +266,6 @@ def expected_settings_names(self):
             'enable_subsection_gating',
             'learning_info',
             'instructor_info',
-            'create_zendesk_tickets',
```

Comment:
These fields do not show for the default bok choy test user. Either the waffle flag needs to be enabled, or the user must be an edX staff/admin user. For this reason, for the purposes of this test, I am removing them.

I am hoping that testing the Python methods that are responsible for whether fields are displayed on the page is sufficient. But if there's a desire to add support for more granular testing of the UI, I can work on that.